framework-reference/store-value.md

```diff
@@ -18,6 +18,7 @@ storeValue(key: string, value: any): void
 | :--- | :--- |
 | **key** | Name of the key to store the value against |
 | **value** | Value to give the key you are creating/updating |
+| **persist**| Should this key value get persisted in the browser local storage to use between sessions. Defaults to true |
```

Comment:
@hetunandu shouldn't we default to false? Can we migrate older apps to a deprecated method or something?

Reply:
Migration would require some script parsing which we can't do today.

Reply:
As discussed, we would tackle this problem for future releases